tests/test_labels.py

```diff
@@ -0,0 +1,37 @@
+"""
+Test if labels are supplied correctly to the container
+"""
+import time
+from repo2docker.app import Repo2Docker
+from repo2docker import __version__
+import pytest
+
+
+@pytest.mark.parametrize('ref', ['some-branch', None])
+def test_labels(ref, tmpdir):
+    app = Repo2Docker()
+    if ref is not None:
+        argv = ['--ref', ref, str(tmpdir)]
+    else:
+        argv = [str(tmpdir)]
```

Comment:
For my education, what does `str(tmpdir)` do or asked differently what is `tmpdir` that we need to call `str` on it?

Reply:
from [the docs](https://docs.pytest.org/en/2.8.7/tmpdir.html):

> tmpdir is a [py.path.local](
http://py.rtfd.org/en/latest/path.html) object which offers os.path methods and more.